client/my-sites/checkout/cart/cart-free-user-plan-upsell.jsx

```diff
@@ -155,10 +156,13 @@ const mapStateToProps = ( state, { cart } ) => {
 		isRegisteringDomain: hasDomainRegistration( cart ),
 		isSitePlansListFetching: isRequestingSitePlans( state ),
 		personalPlan: personalPlan,
-		planPrice: ! isPlansListFetching && getPlanPrice( state, selectedSite.ID, personalPlan, false ),
+		planPrice:
+			! isPlansListFetching &&
+			selectedSiteId &&
+			getPlanPrice( state, selectedSiteId, personalPlan, false ),
```

Comment:
Perhaps it would make sense to fall back to the `getPlanRawPrice` selector for when the site doesn't exist in the current context.

Reply:
This was not supposed to appear in domain only checkout, so I think this will work here.